doc/source/api/index.rst

```diff
@@ -43,6 +43,14 @@ system for reads and variants.
 .. toctree::
    references
 
+Sequence Annotations
```

Comment:
Larger we. Hence `Annotation' should always be qualified
`Sequence Annotations', `Variant Annotations', etc.  I had never
heard variant classification call `variant annotation' until I
showed up at the `Annotation Task Team' call and discovered it
wasn't my type of annotations.

Reece Hart <notifications@github.com> writes:
> In doc/source/api/index.rst:
> 
> > @@ -43,6 +43,14 @@ system for reads and variants.
> >  .. toctree::
> >     references
> >
> > +Sequence Annotations
> 
> Hmm. Okay, this is probably just means that you and I (and maybe the larger we)
> have different colloquial definitions of "annotations" and "features". There's
> probably no right or wrong here, but we should get some consensus on how the
> terms are used. (Ditto for the use of "allele" and "variant"
> semi-interchangeably elsewhere in the schema.)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>